test/include/helpers/commonHelpers.hpp

```diff
@@ -10,6 +10,7 @@
 
 static constexpr float EPSILON_F = 1e-6f;
 static constexpr int maxGPUCoresTestCount = 20000;
+static constexpr float INF = std::numeric_limits<float>::infinity();
```

Comment:
I'd avoid polluting header files with such aliases (especially when it is used in `DistanceFieldTest.cpp` only). I'd move it there.